src/test/kotlin/org/jitsi/nlj/transform/node/incoming/RemoteBandwidthEstimatorTest.kt

```diff
@@ -0,0 +1,106 @@
+package org.jitsi.nlj.transform.node.incoming
+
+import com.nhaarman.mockitokotlin2.spy
+import io.kotlintest.IsolationMode
+import io.kotlintest.Spec
+import io.kotlintest.matchers.numerics.shouldBeGreaterThan
+import io.kotlintest.shouldBe
+import io.kotlintest.shouldNotBe
+import io.kotlintest.specs.ShouldSpec
+import org.jitsi.nlj.PacketInfo
+import org.jitsi.nlj.format.Vp8PayloadType
+import org.jitsi.nlj.format.Vp9PayloadType
+import org.jitsi.nlj.resources.logging.StdoutLogger
+import org.jitsi.nlj.rtp.RtpExtension
+import org.jitsi.nlj.rtp.RtpExtensionType
+import org.jitsi.nlj.test_utils.FakeClock
+import org.jitsi.nlj.test_utils.timeline
+import org.jitsi.nlj.util.StreamInformationStoreImpl
+import org.jitsi.nlj.util.ms
+import org.jitsi.nlj.util.times
+import org.jitsi.rtp.rtp.RtpPacket
+import org.jitsi.rtp.rtp.header_extensions.AbsSendTimeHeaderExtension
+import java.time.Duration
+
+class RemoteBandwidthEstimatorTest : ShouldSpec() {
+    override fun isolationMode(): IsolationMode? = IsolationMode.InstancePerLeaf
+
+    private val clock: FakeClock = spy()
+    private val streamInformationStore = StreamInformationStoreImpl()
+    private val astExtensionId = 3
+    // REMB is enabled by having at least one payload type which has "transport-cc" signaled as a rtcp-fb, and TCC is
+    // disabled.
+    private val vp8PayloadType = Vp8PayloadType(100, emptyMap(), setOf("goog-remb"))
+    private val vp9PayloadTypeWithTcc = Vp9PayloadType(101, emptyMap(), setOf("transport-cc"))
+    private val packetInterval = 10.ms()
+    private val ssrc = 1234L
+
+    private lateinit var remoteBandwidthEstimator: RemoteBandwidthEstimator
+
+    override fun beforeSpec(spec: Spec) {
+        super.beforeSpec(spec)
+        streamInformationStore.addRtpExtensionMapping(RtpExtension(astExtensionId.toByte(), RtpExtensionType.ABS_SEND_TIME))
+        streamInformationStore.addRtpPayloadType(vp8PayloadType)
+        remoteBandwidthEstimator = RemoteBandwidthEstimator(streamInformationStore, StdoutLogger(), clock = clock)
```

Comment:
Pretty sure you could also do all these with the declarations using `apply` i.e.:
```
private val streamInformationStore = StreamInformationStoreImpl().apply {
  addRtpExtensionMapping(RtpExtension(astExtensionId.toByte(), RtpExtensionType.ABS_SEND_TIME))
  addRtpPayloadType(vp8PayloadType)
}

private val remoteBandwidthEstimator = RemoteBandwidthEstimator(streamInformationStore, StdoutLogger(), clock = clock)
```

and get rid of the `beforeSpec` block (would have to move the declaration of the stream information store to after the vp8payload type).